src/encoded/static/components/donor.js

```diff
@@ -3,15 +3,18 @@ import PropTypes from 'prop-types';
 import _ from 'underscore';
 import url from 'url';
 import { Panel, PanelBody, PanelHeading } from '../libs/bootstrap/panel';
+import { auditDecor } from './audit';
 import { ExperimentTable } from './dataset';
 import { DbxrefList } from './dbxref';
 import { DocumentsPanel, DocumentsSubpanels } from './doc';
+import { FetchedData, Param } from './fetched';
```

Comment:
Did you mean to leave this in?  FetchedData, Param are not used in the file.

Reply:
No! I need to remove that.